package.json

```diff
@@ -17,7 +17,7 @@
     "release:github-draft": "./ci/build/release-github-draft.sh",
     "release:github-assets": "./ci/build/release-github-assets.sh",
     "release:prep": "./ci/build/release-prep.sh",
-    "test:e2e": "./ci/dev/test-e2e.sh",
+    "test:e2e": "VSCODE_IPC_HOOK_CLI= ./ci/dev/test-e2e.sh",
```

Comment:
🧹 This is needed if you run the e2e script inside code-server in headfull mode I believe. Otherwise it tries to open the tests in the your running code-server instance.